### DISCUSSION TITLE
Compiling Logging quickstart.cc using g++

### DISCUSSION THREAD

**Discussion by whoop77** [googleapis/google-cloud-cpp#7717] [General]
How to compile Logging quickstart.cc using g++ instead of CMake

**Comment by coryan** [comment]
Maybe this answers your question?

https://github.com/googleapis/google-cloud-cpp/blob/cc5834b632192a6a833e0fb07e45d245fafaa41b/google/cloud/logging/quickstart/Makefile#L27-L35

Where CXXLD is defined as:

https://github.com/googleapis/google-cloud-cpp/blob/cc5834b632192a6a833e0fb07e45d245fafaa41b/google/cloud/logging/quickstart/Makefile#L20-L22

Please do not hard-code the *current* output of `pkg-config --libs-only -l google_cloud_cpp_logging` in your build scripts.  We guarantee to keep this module name (though it is not-quite-GA yet, it is unlikely to change by the time it becomes GA).  We do not guarantee that its output won't change over time.
